scripts/test_in_docker.sh

```
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+export TEST_ARGS="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)